cms-2016-collision-datasets/code/create-das-json-store.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+#
+# Note: to be run as cernapcms on LXPLUS.  Example:
+#
+# cernapcms@lxplus031> mkdir xxx
+# cernapcms@lxplus031> cd xxx
+# cernapcms@lxplus031> voms-proxy-init --voms cms
+# cernapcms@lxplus031> ./create-das-json-store.sh
+#
+# The resulting `*.json` files are to be copied to ../inputs/das-json-store/
+# directory on the working laptop.
+
+for dataset in \
```

Comment:
This for loop can be simplified as follows:

```bash
while IFS= read -r dataset; do
   ...
done < ./inputs/cms-2016-collision-datasets.txt
```

You will not have to list datasets several times then.

For a working example, please see `cms-2012-collision-datasets-update/code/create-das-json-store.sh` or `cms-2015-collision-datasets-hi-ppref/code/create-das-json-config-store.sh` and other such files in this repository.